site/content/docs/traces-open-telemetry/importing-traces/troubleshooting-missing-spans.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Troubleshooting missing or orphan spans
+weight: 20
+menu:
+  platform:
+    parent: "Send your traces to Checkly"
+    identifier: troubleshooting-missing-spans
+navTitle: Missing spans
+beta: true
+---
+
+This guide will walk you through the process of identifying missing spans, understanding their root causes, and implementing solutions to ensure your OpenTelemetry setup with Checkly provides complete and accurate tracing information. 
+
+## Missing spans
+
+Missing spans, also known as orphan spans, are segments of a trace that are disconnected from either their parent spans or from the root span.
+
+An instrumented distributed system, will emit spans creating a hierarchical chain that follows the flow of the request through the different components and services.
+
+When a span is missing, they create a whole in this chain, making it difficult to diagnose issue and understand the full picture.
+
+When viewing trace details, a span might be missing in the Checkly visualization. 
+Using contextual span data, about the trace it belongs to, and it's parent span; Checkly displays any missing pieces on the span chain, so that you can identify which application or service might be incorrectly instrumented. 
+
+![Trace with missing spans](/docs/images/otel/traces-missing-spans.png)
+
+
+## Troubleshooting missing spans
+
+Missing spans can happen due to various reasons.  Wether you're sending data to Checkly directly from your applications and services, or through the OpenTelemetry Collector, these are steps to confirm everything is properly setup.
+
+### Verify sampling strategy
+
+   Using Checkly Traces, we recommend sampling for the tracestate header `checkly=true`, which will reduce your Egress/Ingress costs and ensure that all the spans that were originated through a check are there.
+
+   #### Sampling in the OpenTelemetry Collector
+
+      a. Make sure that you're filtering out non-checkly spans:
```

Comment:
Is this the desired formatting?
<img width="672" alt="Screenshot 2024-10-24 at 08 17 41" src="https://github.com/user-attachments/assets/f00d59d4-0199-48d9-8997-0dac1b5d778e">